apps/ex_nvr/lib/ex_nvr/model/device.ex

```diff
@@ -212,7 +410,8 @@ defmodule ExNVR.Model.Device do
 
   @spec config_updated(t(), t()) :: boolean()
   def config_updated(%__MODULE__{} = device_1, %__MODULE__{} = device_2) do
-    device_1.stream_config != device_2.stream_config or device_1.settings != device_2.settings
+    device_1.stream_config != device_2.stream_config or device_1.settings != device_2.settings or
+      device_1.snapshot_config != device_2.snapshot_config
```

Comment:
I prefer not to restart the pipeline when the snapshot config change. We'll come back to this to find a better solution